.devcontainer/setup-ex.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/python2
+
+# See CONTRIBUTING.md.
+
+from __future__ import print_function, unicode_literals
+import multiprocessing, os, subprocess, time
+
+
+GO_VERSION = os.getenv("GO_VERSION")
```

Comment:
Let's run this inside a function. We should have a single entry function for the script.

Code scattered across the script outside of functions makes it hard to understand control flow for script.

Reply:
it is the same setup.py, I don't want to change it a lot for now.